snyk/managers.py

```diff
@@ -296,6 +296,16 @@ def all(self) -> Any:
 
 
 class IssueSetManager(SingletonManager):
+    def _convert_reserved_words(self, data):
+        if "issues" in data:
```

Comment:
This also need to be extended for `licenses` and not only vulns

Reply:
In master, licenses is still a dict. Should be easy to port in a follow up or the PR that changes licenses to an object?